StyleGuide.md

```diff
@@ -970,6 +975,111 @@ Use the following HTML and Liquid to include an image in a Markdown page:
 
 Example: [Decommission Nodes](https://www.cockroachlabs.com/docs/stable/remove-nodes.html#step-1-check-the-node-before-decommissioning)
 
+<a name="include-files"></a>
+
+### Include files
+
+Sometimes content needs to be duplicated across two or more pages in the documentation. For example, there may be several pages that need the same cluster setup, but describe how to use different features. Or a very specific [note](#notes) or [warning](#warnings) needs to be added to several different pages.
+
+In these situations, you will need to use something we call an _include file_. An include file is a separate Markdown file (stored in `_includes/some/shared-file.md`) where you will write content to be shared across multiple pages.
+
+For more information about include files, see [the Jekyll `include` documentation](https://jekyllrb.com/docs/includes/).
+
+_Note_: Using include files adds complexity to the docs site architecture and build process. It also makes writing the documentation more tricky. If you can link to existing content rather than using an include file, strongly consider doing that instead.
+
+There are [basic](#basic-include-file-usage) and [advanced](#advanced-include-file-usage) methods for using include files. Use the basic method unless you are sure you need the advanced.
+
+<a name="basic-include-file-usage"></a>
+
+#### Basic include file usage
+
+The basic method for using an include file is:
+
+1. Find (or create) a block of content that you want to make appear on two or more different pages.
+2. Create a new file in the subdirectory of the `_includes` directory associated with the product you are working on. For example, if you are working on CockroachDB Self-Hosted v22.1, the directory will be `_includes/v22.1/`. If you are working on CockroachDB Dedicated, it will be `_includes/cockroachcloud`. If the include text is not version- or product-specific, it can go in the `common` folder.
+3. From the pages where you want the included content to appear, add an `include` tag that looks like one of the following: for CockroachDB Self-Hosted, it will look like `{% include {{page.version.version}}/some/shared-file.md %}`; for CockroachDB Dedicated, it will look like `{% include cockroachcloud/some/shared-file.md %}`.
```

Comment:
```suggestion
1. From the pages where you want the included content to appear, add an `include` tag that looks like one of the following: for CockroachDB Self-Hosted, it will look like `{% include {{page.version.version}}/some/shared-file.md %}`; for CockroachDB Dedicated, it will look like `{% include cockroachcloud/some/shared-file.md %}`.
```